README.md

```diff
@@ -74,7 +55,12 @@ tar -xzf data.tar.gz
 We will update the snapshot once a day. If you plan to run your node as a [gateway](#gateway), please use the snapshot with index data:
 https://t.iotex.me/mainnet-data-with-idx-latest.
 
-(Optional) If you want to sync the chain from 0 height, please change the `gravityChainAPIs` in config.yaml to use your infura key with archieve mode supported or change the API endpoint to an Ethereum node with archieve data which you can access.
+(Optional) If you only want to sync chain data from 0 height without relaying on legacy delegate election data from Ethereum, you can setup legacy delegat election data with following command:
+```bash
+curl -L https://storage.googleapis.com/blockchain-golden/poll.mainnet.tar.gz > $IOTEX_HOME/poll.tar.gz; tar -xzf $IOTEX_HOME/poll.tar.gz --directory $IOTEX_HOME/data
+```
+
+(Optional) If you want to sync the chain from 0 height and also fetching legacy delegate election data from Ethererum, please change the `gravityChainAPIs` in config.yaml to use your infura key with archieve mode supported or change the API endpoint to an Ethereum node with archieve data which you can access.
```

Comment:
1. archieve --> archive
2. elaborate on how to enable "with archive mode supported"?

Reply:
enable ethererum archive mode is an ethererum thing. people should find it in ethererum or infura doc